book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
+
```

Comment:
- [ ] I would recommend you move this code to the top of the class.
- [ ] Also, kindly add `:rentals` to your `attr_accessor` declaration so that you would be able to access and add a rental from your Rental class.